### DISCUSSION TITLE
Tomb Raider 2013 - Multiplayer PC

### DISCUSSION THREAD

**Discussion by tatehira** [community/community#68099] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys, i wanHey guys! I played this game "Tomb Raider 2013", there are several people who managed to change the position in the multiplayer rankings, if you enter the rankings, you will see that there are many people who have 99999 kills and 0 deaths. I'm a Cyber ​​Security student and one of my goals is to be able to do this, it's a childhood dream. It is not possible to do this via the cheat engine, I was only able to increase the player's level online. Does anyone have tips on how I can change my position in the ranking?

**Comment by yalza** [comment]
Response #1
